engine/src/shell/platform/unity/uiwidgets_panel.cc

```diff
@@ -412,6 +417,25 @@ void UIWidgetsPanel::OnMouseLeave() {
   }
 }
 
+void UIWidgetsPanel::TakeScreenShot() {
+  if (process_events_) {
+    auto* engine = reinterpret_cast<EmbedderEngine*>(engine_);
+    Rasterizer::Screenshot screenshot = engine->GetShell().Screenshot(
+        Rasterizer::ScreenshotType::SkiaPicture, true);
+    const char* screenshot_char =
+        static_cast<const char*>(screenshot.data->data());
+    int size = screenshot.data->size();
+    SkBase64 temp;
+    temp.decode(screenshot_char, screenshot.data->size());
+    char* decoded = temp.getData();
+
+    std::ofstream myfile;
+    myfile.open("example.txt");
```

Comment:
this api is not good designed. saving the screenshot to a magic path doesn't make any sense to me. to my understanding, it is only used for internal debug